src/plugins/dashboard/public/dashboard_container/embeddable/create/controls/dashboard_control_group_integration.test.ts

```diff
@@ -11,7 +11,7 @@ import { Filter } from '@kbn/es-query';
 import { combineDashboardFiltersWithControlGroupFilters } from './dashboard_control_group_integration';
 import { BehaviorSubject } from 'rxjs';
 
-jest.mock('@kbn/controls-plugin/public/react_controls/control_group/get_control_group_factory');
+jest.mock('@kbn/controls-plugin/public/control_group/get_control_group_factory');
```

Comment:
I don't think this mock is needed for the test. You should be able to remove this.

Reply:
Good catch :+1: